src/Interfaces/Client/ClientInterface.php

```diff
@@ -19,6 +19,8 @@
 use TrueLayer\Interfaces\Payment\PaymentCreatedInterface;
```

Comment:
dump question: why the ClientInterface does not extends `PaymentsApiInterface`? it seems to me very close as a type